### PR TITLE
Add support for Power machines

### PR DESCRIPTION
**Pull request by arbulu89**
https://trello.com/c/QFlTJfWU/768-add-dynamic-path-search-for-hdbclm-command
https://trello.com/c/AlM5SCiB/765-add-support-for-power-machines

### CODE REVIEW

File: shaptools/hana.py [hunk 1]
```diff
@@ -591,10 +613,11 @@ def set_ini_parameter(
         user_name = kwargs.get('user_name', None)
         user_password = kwargs.get('user_password', None)
 
-        self._manage_ini_file(parameter_str=parameter_str, database=database,
-                              file_name=file_name, layer=layer, layer_name=layer_name,
-                              set_value=True, reconfig=reconfig, key_name=key_name,
-                              user_name=user_name, user_password=user_password)
+        self._manage_ini_file(
```
Comment (MalloZup) [inline shaptools/hana.py]: why are you adding a newline here? I mean seems unrelated to PR IMHO :grin:

Comment (arbulu89) [reply shaptools/hana.py]: Yes... Just it looks like much better this way (following python patterns). Otherwise there are lines with spaces instead of tabs...

**Comment by arbulu89** [comment]
Tested and working for intel machines

File: shaptools/hana.py [hunk 2]
```diff
@@ -66,7 +67,12 @@ class HanaInstance:
     """
 
     PATH = '/usr/sap/{sid}/HDB{inst}/'
-    INSTALL_EXEC = '{software_path}/DATA_UNITS/HDB_LCM_LINUX_X86_64/hdblcm'
+    INSTALL_EXEC = '{software_path}/DATA_UNITS/{platform}/hdblcm'
+    PLATFORMS = {
```
Comment (diegoakechi) [inline shaptools/hana.py]: See: 
https://github.com/SUSE/saphanabootstrap-formula/pull/45#discussion_r302940120
